src/set_theory/cardinal/basic.lean

```diff
@@ -874,35 +867,39 @@ nat.cast_injective
 
 @[simp] theorem succ_zero : succ (0 : cardinal) = 1 := by norm_cast
 
-theorem card_le_of {α : Type u} {n : ℕ} (H : ∀ s : finset α, s.card ≤ n) :
-  # α ≤ n :=
+theorem card_le_of {α : Type u} {n : ℕ} (H : ∀ s : finset α, s.card ≤ n) : # α ≤ n :=
 begin
   refine le_of_lt_succ (lt_of_not_ge $ λ hn, _),
-  rw [← cardinal.nat_succ, ← cardinal.lift_mk_fin n.succ] at hn,
+  rw [←cardinal.nat_succ, ←lift_mk_fin n.succ] at hn,
   cases hn with f,
-  refine not_lt_of_le (H $ finset.univ.map f) _,
-  rw [finset.card_map, ← fintype.card, fintype.card_ulift, fintype.card_fin],
+  refine (H $ finset.univ.map f).not_lt _,
+  rw [finset.card_map, ←fintype.card, fintype.card_ulift, fintype.card_fin],
   exact n.lt_succ_self
 end
 
 theorem cantor' (a) {b : cardinal} (hb : 1 < b) : a < b ^ a :=
-by rw [← succ_le_iff, (by norm_cast : succ (1 : cardinal) = 2)] at hb;
-   exact (cantor a).trans_le (power_le_power_right hb)
+begin
+  rw [←succ_le_iff, (by norm_cast : succ (1 : cardinal) = 2)] at hb,
```

Comment:
```suggestion
  rw [←succ_le_iff, by norm_cast : succ (1 : cardinal) = 2] at hb,
```

Reply:
This one seems to have broken.

Reply:
sorry for that 😅 I should have warned they were untested.